### ISSUE TITLE
Which models support additional features?

### ISSUE THREAD

**Issue by rmitsch** [awslabs/gluon-ts#781]
Which currently implemented models support additional features, i. e. not just the past values of the target time series, but also `feat_dynamic_real`, `feat_dynamic_static`, `cat_static_real`? 
E. g. DeepState and DeepAR do, other models don't seem to. I wasn't able to find a concise documentation on that though.

**Comment by timoschowski** [comment]
yes, I don't think we have this documentation at the moment, I'm afraid. Is this something you'd be open to contribute back? On rough terms: essentially all deep learning models should be able to handle this, ARIMA and Prophet as well, ETS not.
